rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -442,6 +443,66 @@ def test_binance_api_without_markets_error(rotkehlchen_api_server: 'APIServer')
     assert_simple_ok_response(response)
 
 
+def test_kraken_futures_only_one_key(rotkehlchen_api_server: 'APIServer') -> None:
+    """
+    Test that adding or editing only one of the 2 required
+    API keys for Kraken Futures returns error
+    """
+    _kraken_one_key_test_logic(rotkehlchen_api_server, requests.put)
```

Comment:
i don't think you need the function. you can easily make it a for-loop with various conditions and expected result